prelude/__internal__/Dictionary.mad

```diff
@@ -26,7 +26,7 @@ export fromList = pipe(
 
 
 /**
- * The empty Dictionary
+ * Creates an empty Dictionary.
```

Comment:
Actually it doesn't really create anything, it's already created, it's just a constant in that sense.

Reply:
Yeah, but I think that level of specificity isn't really important? In my mind if I call a function and it returns an empty array, I don't really care whether it's a new array or some constant `THE_LAST_LIVING_LIST`, if at the end of it I get back an empty List.

Reply:
What I mean is, it's not a function.

Reply:
Like in the type annotation there’s no arrow. It’s not something you call. It’s very much just a value. So in a program the empty dictionary will always have the same reference, and any transformation of it will always return a new reference.

Reply:
Ohhhhh. Well then I would push for not even documenting it for now.

Reply:
Also the empty value has mathematic meaning and Monoid is there to define mempty. Dictionary doesn’t implement it yet but it will for sure. With that in mind I’d still document it. It is exported and is important imo.

Reply:
Restored to the original docs.